src/App.js

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
+import { createGlobalStyle } from 'styled-components'
 import './App.css';
 import MentorMe from './views/MentorMe';
 
 function App() {
   return (
     <div className="App">
+    <GlobalStyles />
```

Comment:
Doesn't this need to wrap the App component (or MentorME comp.)?

Reply:
It's a self-closing tag that applies global styles to all components. It doesn't have to wrap any components